### PR TITLE
feature/register-style

### PR DESCRIPTION
**Pull request by Eddie98Lopez**
## Description

Register route is now responsive to changes in viewport width. Styles mimic the login page. This change was only to modify styles and should not modify functionality. 

## Type of change

Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] This change requires a documentation update

## Checklist:

- [ ] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my own code
- [ ] I have removed unnecessary comments/console logs from my code
- [ ] I have made corresponding changes to the documentation if necessary (optional)
- [ ] My changes generate no new warnings
- [ ] I have checked my code and corrected any misspellings
- [ ] No duplicate code left within changed files
- [ ] Size of pull request kept to a minimum
- [ ] Pull request description clearly describes changes made & motivations for said changes

### CODE REVIEW

**Review by salvare736 — APPROVED** [review]
Looks good, Lalo. Thank you!

**Review by elCarlosSantiago — APPROVED** [review]
Clean work Lalo!

File: src/components/pages/Login/SignupPage.js [hunk 1]
```diff
@@ -5,6 +5,10 @@ import logo from '../../../styles/hrf-logo.png';
 import axios from 'axios';
 import { notification } from 'antd';
 import { CheckCircleOutlined, CloseCircleOutlined } from '@ant-design/icons';
+import blue from '../../../styles/blue.svg';
+import blue1 from '../../../styles/blue1.svg';
+import blue2 from '../../../styles/blue2.svg';
+import liberty from '../../../styles/liberty.png';
```
Comment (ryan-hamblin) [inline src/components/pages/Login/SignupPage.js:11]: Interested in the break in pattern here? Why .png not .svg?

File: src/components/pages/Login/_SignupPageStyled.less [hunk 2]
```diff
@@ -5,40 +5,121 @@
 @font: 'Lato', sans-serif;
 
 .signup-root {
-  width: 100%;
+  width: 100vw;
+  height: 100vh;
+  overflow:hidden;
```
Comment (ryan-hamblin) [inline src/components/pages/Login/_SignupPageStyled.less:10]: add space after `:`

File: src/components/pages/Login/_SignupPageStyled.less [hunk 3]
```diff
@@ -5,40 +5,121 @@
...
   display: flex;
-  background-image: url('../../../styles/blueStatueOfLiberty.png');
-  background-size: cover;
-  background-repeat: no-repeat;
+  justify-content: center;
+  align-items: center; 
+}
+
+.background{
+  width:100%;
```
Comment (ryan-hamblin) [inline src/components/pages/Login/_SignupPageStyled.less:18]: spacing after the `:`'s is inconsistant. Let's standardize and pick a pattern.

File: src/components/pages/Login/_SignupPageStyled.less [hunk 4]
```diff
@@ -5,40 +5,121 @@
...
+  height:100vh;
+  z-index:-3;
+  position: relative;
+}
+
+.svgContain{
```
Comment (ryan-hamblin) [inline src/components/pages/Login/_SignupPageStyled.less:24]: same here, spacing before the `{` is inconsistent with other property declarations.

File: src/components/pages/Login/_SignupPageStyled.less [hunk 5]
```diff
@@ -5,40 +5,121 @@
...
+  height:100%;
+
+  @media (max-width: 768px) {
+    display:none;
+  }
+
+}
+
+.blueShape1{
+
```
Comment (ryan-hamblin) [inline src/components/pages/Login/_SignupPageStyled.less:42]: This spacing seems unecessary

**Pull request merged**